Sources/SpeziScheduler/Notifications/SchedulerNotifications.swift

```diff
@@ -24,6 +24,48 @@ import struct SwiftUI.AppStorage
 ///
 /// Notifications can be automatically scheduled for Tasks that are scheduled using the ``Scheduler`` module. You configure a Task for automatic notification scheduling by
 /// setting the ``Task/scheduleNotifications`` property.
+/// 
+/// When notifications are enabled, the app will send alerts based on the task’s schedule. 
+///     If you want to stop notifications for a specific event, you can call ``Event/complete()`` to mark the event as complete.
+///     This will prevent notifications only for that particular occurrence, while other events will still trigger notifications as scheduled.
+///
+/// Below is an example of how to schedule a task with notifications and selectively disable notifications for certain events:
```

Comment:
this is a great addition!

hopefully at some point in the future we might have some kind of dedicated API for dealing with this use case, but until then it's really nice to have this approach documented...